### PR TITLE
Update release notes for v1.6.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@rhatdan @baude @TomSweeneyRedHat @haircommander PTAL

**Comment by mheon** [comment]
@QiWang19 a PR like this is the first step in the Podman release process. We look at all the PRs tagged with the appropriate label (`Release Notes 1.6.0` for this release) that were merged, and then compile them all in `RELEASE_NOTES.md`. Anything not yet merged will be untagged 1.6.0.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
 # Release Notes
 
+## 1.6.0
+### Features
+- The `podman network create`, `podman network rm`, `podman network inspect`, and `podman network ls` commands have been added to manage CNI networks used by Podman
+- The `podman volume create` command can now create and mount volumes with options, allowing volumes backed by NFS, tmpfs, and many other filesystems
+- Podman can now run containers without CGroups for better integration with systemd via the `--cgroups=disabled` flag to `podman create` and `podman run`. This is presently only supported with the `crun` OCI runtime
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: suggest "systemd by using the `--cgroups=disabled` option with the `podman create` and `podman run` commands"

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
...
+- The `podman volume rm` and `podman volume inspect` commands can now refer to volumes by unambiguous partial name, in addition to full name (e.g. `podman volume rm myvol` to remove a volume named `myvolume`) ([#3891](https://github.com/containers/libpod/issues/3891))
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: s/an unambiguous/unambigous

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+- The `podman run` and `podman create` commands now support the `--pull` flag to allow forced re-pulling of images ([#3734](https://github.com/containers/libpod/issues/3734))
+- Mounting volumes into a container using `--volume`, `--mount`, and `--tmpfs` now allows the `suid`, `dev`, and `exec` mount options (the inverse of `nosuid`, `nodev`, `noexec`) ([#3819](https://github.com/containers/libpod/issues/3819))
+- The `podman push` command now supports the `--digestfile` option to save a file containing the pushed digest
+- Pods can now have their hostname set via `podman pod create --hostname` or providing Pod YAML with a hostname set to `podman play kube` ([#3732](https://github.com/containers/libpod/issues/3732))
+- The `podman image sign` command now supports the `--cert-dir` flag
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: s/option/flag

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
+- The `podman run` and `podman create` commands now support the `--security-opt label=filetype:$LABEL` flag to set the SELinux label for container files
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: s/option/flag

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -1,5 +1,57 @@
...
+- Fixed a bug where improper permissions on the `~/.config` directory could cause rootless Podman to use an incorrect directory for storing some files
+- Fixed a bug where the bash completions for `podman import` threw errors
+- Fixed a bug where Podman volumes created with `podman volume create` would not copy the contents of their mountpoint the first time they were mounted into a container ([#3945](https://github.com/containers/libpod/issues/3945))
+- Fixed a bug where rootless Podman could not run `podman exec` when the container was not run inside a CGroup owned by the user ([#3937](https://github.com/containers/libpod/issues/3937))
+- Fixed a bug where `podman play kube` would panic when given Pod YAML without a `securityContext` ([#3956](https://github.com/containers/libpod/issues/3956))
+- Fixed a bug where Podman would place files incorrectly when `storage.conf` configuration items were set to the empty string ([#3952](https://github.com/containers/libpod/issues/3952))
+- Fixed a bug where `podman build` did not correctly inherit Podman's CGroup configuration, causing crashed on CGroups V2 systems ([#3938](https://github.com/containers/libpod/issues/3938))
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: Based on what I found in https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html, I'm never capitalizing 'cgroup' and never pluralizing it.  YMMV.

**Comment by mheon** [comment]
I think I'm going to keep `flag` vs `option` and CGroup vs cgroup for now; they're used in older releases too.

We might want to go in with sed and hit all of them at once, though.

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -1,5 +1,57 @@
...
+- Fixed a bug where temporary files were not properly cleaned up after a build using remote Podman ([#3869](https://github.com/containers/libpod/issues/3869))
+- Fixed a bug where `podman remote cp` crashed instead of reporting it was not yet supported ([#3861](https://github.com/containers/libpod/issues/3861))
+- Fixed a bug where `podman exec` would run as the wrong user when execing into a container was started from an image with Dockerfile `USER` (or a user specified via `podman run --user`) ([#3838](https://github.com/containers/libpod/issues/3838))
+- Fixed a bug where images pulled using the `oci:` transport would be improperly named
+- Fixed a bug where `podman varlink` would hang when managed by systemd due to SD_NOTIFY support conflicting with Varlink ([#3572](https://github.com/containers/libpod/issues/3572))
+
+### Misc
+- Significant changes were made to Podman volumes in this release. If you have pre-existing volumes, it is strongly recommended to run `podman system renumber` after upgrading.
+- Podman now uses the CNI firewall plugin, requiring at least version 0.8.1 of the CNI plugins are installed
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: I'm not sure what you're trying to say after the comma here?  Perhaps "requiring at least version 0.8.1 for the CNI plugins to be installed"

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -1,5 +1,57 @@
...
+- Version 2.0.1 of Conmon is strongly recommended
+- Updated vendored Buildah to v1.11.2
+- Improved error messages when trying to run `podman pause` or `podman stats` on a rootless container on a system without CGroups V2 enabled
+- `TMPDIR` has been set to `/var/tmp` by default to better handle large temporary files
+- Waiting for containers has been optimized to detect container stop more rapidly
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: suggest "container stop" to "container stoppage" or perhaps "that the container has stopped"

File: RELEASE_NOTES.md [hunk 8]
```diff
@@ -1,5 +1,57 @@
...
+- Podman containers now include a `ContainerManager` annotation indicating they were created by `libpod`.
+- The `podman info` command now includes information about `slirp4netns` and `fuse-overlayfs` if they are available
+- Podman no longer sets a default size of 65kb for tmpfs filesystems
+- The default Podman CNI network has been renamed in an attempt to prevent conflicts with CRI-O when both are run on the same system. This should only take effect on system restart.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: I luv the ending period, but you don't seem to have them elsewhere!   sigh

**Comment by mheon** [comment]
@rhatdan @baude Anyone want to whack this with a /lgtm /hold?

File: RELEASE_NOTES.md [hunk 9]
```diff
@@ -1,5 +1,57 @@
...
+- Fixed a bug where improper permissions on the `~/.config` directory could cause rootless Podman to use an incorrect directory for storing some files
+- Fixed a bug where the bash completions for `podman import` threw errors
+- Fixed a bug where Podman volumes created with `podman volume create` would not copy the contents of their mountpoint the first time they were mounted into a container ([#3945](https://github.com/containers/libpod/issues/3945))
+- Fixed a bug where rootless Podman could not run `podman exec` when the container was not run inside a CGroup owned by the user ([#3937](https://github.com/containers/libpod/issues/3937))
+- Fixed a bug where `podman play kube` would panic when given Pod YAML without a `securityContext` ([#3956](https://github.com/containers/libpod/issues/3956))
+- Fixed a bug where Podman would place files incorrectly when `storage.conf` configuration items were set to the empty string ([#3952](https://github.com/containers/libpod/issues/3952))
+- Fixed a bug where `podman build` did not correctly inherit Podman's CGroup configuration, causing crashed on CGroups V2 systems ([#3938](https://github.com/containers/libpod/issues/3938))
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: sorry , just spotted.  s/crashes/crashed/

File: RELEASE_NOTES.md [hunk 10]
```diff
@@ -1,5 +1,57 @@
...
+- Fixed a bug where the `commit` Varlink API would segfault if provided incorrect arguments ([#3897](https://github.com/containers/libpod/issues/3897))
+- Fixed a bug where temporary files were not properly cleaned up after a build using remote Podman ([#3869](https://github.com/containers/libpod/issues/3869))
+- Fixed a bug where `podman remote cp` crashed instead of reporting it was not yet supported ([#3861](https://github.com/containers/libpod/issues/3861))
+- Fixed a bug where `podman exec` would run as the wrong user when execing into a container was started from an image with Dockerfile `USER` (or a user specified via `podman run --user`) ([#3838](https://github.com/containers/libpod/issues/3838))
+- Fixed a bug where images pulled using the `oci:` transport would be improperly named
+- Fixed a bug where `podman varlink` would hang when managed by systemd due to SD_NOTIFY support conflicting with Varlink ([#3572](https://github.com/containers/libpod/issues/3572))
+
+### Misc
+- Significant changes were made to Podman volumes in this release. If you have pre-existing volumes, it is strongly recommended to run `podman system renumber` after upgrading.
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md]: ending period.

**Comment by TomSweeneyRedHat** [comment]
Just found two more nits.  Feel free to ignore them, but if by chance you need to make other changes, please nab them.

**Comment by mheon** [comment]
@baude @rhatdan I think CI on master is broken - the F29 remote client system tests seem to be failing everywhere

**Pull request merged**